node/server.js

```diff
@@ -5585,6 +5585,50 @@ function init_io() {
 			resend(player, "reopen+nc+inv");
 			success_response({ num: num, cevent: true });
 		});
+
+		socket.on("destat", function (data) {
+			var player = players[socket.id];
+			var item = player.items[data.item_num];
```

Comment:
The variable was `num` in the client-side function.

Reply:
Fixed.